appstudio-controller/controllers/appstudio.redhat.com/deploymenttargetbinder_controller.go

```diff
@@ -85,30 +85,57 @@ func (r *DeploymentTargetClaimReconciler) Reconcile(ctx context.Context, req ctr
 	// Handle deletion if the DTC has a deletion timestamp set.
 	if dtc.GetDeletionTimestamp() != nil {
 		log.Info("Handling a deleted DeploymentTargetClaim")
-		// If the DTC is bound set the status of the corresponding DT to Released
+		// If the DTC is bound set, handle the corresponding DT
 		if isDTCBound(dtc) {
 			dt, err := getDTBoundByDTC(ctx, r.Client, &dtc)
 			if err != nil && !apierr.IsNotFound(err) {
 				return ctrl.Result{}, err
 			}
 
 			if dt != nil {
-				err = updateDTStatusPhase(ctx, r.Client, dt, applicationv1alpha1.DeploymentTargetPhase_Released, log)
-				if err != nil {
+				var dtcls *applicationv1alpha1.DeploymentTargetClass
+				if dtcls, err = findMatchingDTClassForDT(ctx, r.Client, *dt); err != nil {
 					return ctrl.Result{}, err
 				}
-				log.Info("DeploymentTarget is released", "DeploymentTarget", dt.Name)
```

Comment:
Do we want to also log this after DT is set to the `Released` phase?

Reply:
No, thx